guides/common/attributes.adoc

```diff
@@ -239,7 +239,7 @@ ifeval::["{build}" == "foreman-deb"]
 :Team: Foreman developers
 :project-client-RHEL7-url: https://yum.theforeman.org/client/{ProjectVersion}/el7/x86_64/foreman-client-release.rpm
 :project-client-name: https://yum.theforeman.org/client/{ProjectVersion}/
-:BaseURL: https://docs.theforeman.org/master/
+:BaseURL: https://docs.theforeman.org/nightly/
 :ManagingHostsDocURL: {BaseURL}Managing_Hosts/index-foreman.html#
```

Comment:
These are not rendering for me. 
They should fail but I get redirected to Satellite docs. 
Other than that everything looks fine.

Reply:
I don't understand. I just pasted the output of the htmlvalidation from CI and it fails, but this is expected. Until we merge this, nothing is present.